js/src/setup-mc/setup-stepper/store-requirements/pre-launch-checklist/index.scss

```diff
@@ -8,4 +8,9 @@
 			vertical-align: text-bottom;
 		}
 	}
+
+	.gla-pre-launch-checklist__checkbox {
+		display: flex;
+		justify-content: stretch;
+	}
```

Comment:
It would be great if this style is placed along with the corresponding component's file - js/src/components/pre-launch-check-item/index.scss